src/config.yaml

```diff
@@ -9,14 +9,17 @@ options:
     type: string
     default: ""
     description: |
-      Accepted values are s3 | scp | sftp | ftp | rsync | file
+      Accepted values are s3 | scp | sftp | ftp | rsync | file | azure
       An empty string will disable backups.
   remote_backup_url:
     type: string
     default: ""
     description: |
       URL to the remote server and its local path to be used as the
-      backup destination.
+      backup destination. 
+      While using azure as a backend:
+      Duplicity will take care to create the container 
+      when performing the backup.  Do not create it manually before.
```

Comment:
not a problem, it will start putting data inside it, but if you explicitly write the name for example: azure:///some-dir
then it would throw an error, invalid name because 
duplicity add the unit name as container-name. so now the backed url will become - azure:///some-dir/duplicity-<id> and the error would be : "Invalid Azure Storage Blob container name"

so if container is already created on azure then it would be fine. but we should not explicitly set the name in the config.